idunn/blocks/opening_hour.py

```diff
@@ -40,6 +37,17 @@ def get_tz(es_poi):
             return timezone(tz.tzNameAt(lat, lon))
     return None
 
+
+class OpeningHoursType(types.Type):
+    beginning = validators.String()
+    end = validators.String()
+
+class DaysType(types.Type):
+    dayofweek = validators.Integer(minimum=1, maximum=7)
+    local_date = validators.Date()
+    status = validators.String(enum=['open', 'closed'])
+    opening_hours = validators.Array(items=OpeningHoursType)
+
 class OpeningHourBlock(BaseBlock):
     BLOCK_TYPE = 'opening_hours'
```

Comment:
Minor observation (without link with the PR) but double quote here (ie "opening_hours") is probably better.